alf/environments/fast_parallel_environment.py

```diff
@@ -188,6 +188,16 @@ def num_tasks(self):
     def task_names(self):
         return self._task_names
 
+    def set_commands(self, commands):
```

Comment:
I suggest to add a generic call function to call each parallel environment with same arguments:
```
def call_with_same_args(self, func_name, *args, **kwargs);
```

Reply:
Good point.  Done for both PRs.  PTAL.